### DISCUSSION TITLE
VaultWarden - 1.25.1. An unexpected error has occured during user confirmation

### DISCUSSION THREAD

**Discussion by Untouchamp** [dani-garcia/vaultwarden#3428] [Q&A]
![image](https://user-images.githubusercontent.com/87672816/230625719-b89175eb-5904-4dc0-861b-c3875cc95ca9.png)
Hello, faced this error during confirmation of the user that accepted the invitation.

**Environment:**
Bitwarden system is hosting on our own machines. VaultWarden - 1.25.1

**Precondition:**
There is at least one organization in the bitwarden system that has invited user that accepted the invitation. Current user should have owner role in that organization.

**Steps:**

1. Go to the organization page;
2. Switch to the Manage tab;
3. Hover over user that has accepted invitation;
4. Click on gear icon;
5. Click on "Confirm" option;
6. In the opened modal window click "Confirm";

**Actual result:**
Confirmation operation wasn't completed, no request was send. 
There were two errors in the console (shown on the screenshot). One of them was: Received API exception: DataError.

**Note:**
During debugging it seems like system uses old key that is invalid, so we can't event reach request sending. Invitation was outdated maybe? Those invitation on the screenshot was made 04/06/2023 and error occured 04/07/2023.

**Comment by BlackDex** [comment]
For one, i suggest to update to the latest stable version of Vaultwarden v1.28.1.
Second, an invite is only valid for 5 days (default), but can be configured in newer versions of Vaultwarden.

Also, if you somehow redeployed the container, and removed the key files, those invites are not valid anymore at all, since the key files in the data folder are used to generate the JWT and validate them.
